test/accountSpec.js

```diff
@@ -632,6 +630,86 @@ describe('Accounts', function () {
       })
     })
 
+    it('should resend notifications when asked to', function * () {
+      const listener = sinon.spy()
+      this.socket.on('message', (msg) => listener(JSON.parse(msg)))
+
+      const transfer = this.transfer
+
+      yield this.request()
+        .put(transfer.id)
+        .auth('alice', 'alice')
+        .send(transfer)
+        .expect(201)
+        .expect(validator.validateTransfer)
+        .end()
+
+      // TODO: Is there a more elegant way?
+      yield timingHelper.sleep(50)
+
+      const transferId = transfer.id.match('http://localhost/transfers/(.+)')[1]
+      this.socket.send(transferId)
```

Comment:
I think you should send some kind of request type, not just the transferId (in case you want to add other types of messages in the future):

```js
{
  type: 'request_notification',
  id: '...'
}
```

or something like that